WordPress/Classes/Services/MediaService.m

```diff
@@ -213,16 +213,22 @@ - (void)getMediaURLFromVideoPressID:(NSString *)videoPressID
                             success:(void (^)(NSString *videoURL, NSString *posterURL))success
                             failure:(void (^)(NSError *error))failure
 {
-    NSSet *mediaSet = [blog.media filteredSetUsingPredicate:[NSPredicate predicateWithFormat:@"shortcode = %@", videoPressID]];
+    NSSet *mediaSet = [blog.media filteredSetUsingPredicate:[NSPredicate predicateWithFormat:@"videopressGUID = %@", videoPressID]];
```

Comment:
So I know this is how it was done before but I'd rather we query Core Data for the media items with a `NSFetchRequest`. I could see this getting strange if we got to the point of syncing all media (or maybe we do) and this causes a lot of faults to fill since its post-Core Data.